### PR TITLE
UI optimisations and minor logical adjustment

### PR DESCRIPTION
**Pull request by blueset**
* Update school selector filter in "shared/components/sign" to follow the current year instead of hardcoded year numbers.
* Optimised list view of Paths, Levels, and Problems for small view 
 * removed level language tag text
 * added smaller icon of language in front
 * changed icon for "Run" button in small view
* UI optimisation: 
 * changed icon of New Path from `add-circle-outline` to `add`
 * changed nav-bar button from `md-raised md-primary` to "flat" button
 * Added style for school without second student in `classmentors/components/ace`
 * Fixed phrasing in alert text in `setCodeCombatUserIdCtrlInitialData`
 * Changed buttons in creating new event in ClassMentors from `raised-primary`and `raised` to `primary` and "flat"
 * Changed all `add-circle-outline` icon to `add` icon
 * Fixed alignments of names in Event Participants Challenges / Ranking View.
 * Changed leaving group icon from `highlight-remove` to `clear`
 * Trying to fix icon overlap issue by changing `row-height` to 3:4 ratio. ( issue #54 )

### CODE REVIEW

File: src/shared/components/sign/sign.js [hunk 1]
```diff
@@ -33,6 +33,9 @@
       if ($scope.currentUser.school) {
         $scope.currentUser.school = SPF_SINGAPORE_SCHOOLS[$scope.currentUser.school.id];
       }
+
+      // Current year used to calculate age
+      $scope.currentYear = new Date().getFullYear();
     }
```
Comment (dinoboff) [inline src/shared/components/sign/sign.js]: @blueset @SingaporeClouds 

The year of reference should be the current term year, not such the current year. When does it start in Singapore?

File: src/classmentors/components/events/events.js [hunk 2]
```diff
@@ -146,7 +146,7 @@
         opts.push({
           title: 'New event',
           url: '#' + urlFor('newEvent'),
-          icon: 'add-circle-outline'
+          icon: 'add'
```
Comment (dinoboff) [inline src/classmentors/components/events/events.js]: I can't find any 'add' icon.

**Comment by dinoboff** [comment]
When merging it locally, I can't see the list of path except for the language icon.

After resolving the conflict and reverting the 'add' icon, I pushed it to the staging version (using an always empty DB):

http://firebase-dev.singpath-hd.appspot.com/#/paths
http://firebase-dev.engagementplatform.appspot.com/#/ace-of-coders

I will have better look later this evening. Thanks.

**Comment by blueset** [comment]
@dinoboff 
Sorry for the additional bug introduced. I didn't managed to test this commit locally as I didn't managed to create a local testing environment. 
If possible, I would like to know how to clone a similar development environment locally. Thank you.

File: src/shared/components/sign/sign.js (hunk 1 above)
Comment (blueset) [inline src/shared/components/sign/sign.js]: As far as I know, secondary schools, junior colleges and polytechnics start their term on the first working day of January.

**Comment by dinoboff** [comment]
@blueset I will update the README and fix/update mockups of the backend.

**Comment by blueset** [comment]
@dinoboff Mac OS X 10.10 with Docker and Windows 10 VM.

**Pull request closed without merging**